### ISSUE TITLE
PSReadline 2.1.0 doesn't work in the windows virtual desktop web interface

### ISSUE THREAD

**Issue by jszabo98** [PowerShell/PSReadLine#2725]
## Environment

```none
PS version: 7.1.3
PSReadline version:
os: 10.0.19041.1 (WinBuild.160101.0800)
PS file version: 7.1.3.0
HostName: ConsoleHost
BufferWidth: 120
BufferHeight: 9001
```

## Steps to reproduce

It's simply that you can't type in powershell with the psreadline 2.1.0 module loaded in the wvd web interface or remote desktop web client, or in pwsh at all.

**Comment by daxian-dbw** [comment]
@theJasonHelmick Can you please try out the windows virtual desktop web interface and see how PowerShell works with it?

**Comment by jszabo98** [comment]
It works with powershell 7 and psreadline version 2.2.0-beta4 in wvd in a browser.

**Comment by daxian-dbw** [comment]
@jszabo98 Thanks for the update! I'm having trouble creating an Azure Virtual Desktop myself and thus haven't verified the reported issue yet.

> It works with powershell 7 and psreadline version 2.2.0-beta4 in wvd in a browser.

Did you try with PowerShell 7.2 + PSReadLine 2.2.0-beta4? Can you please share a screenshot of using it in a web client?

**Comment by jszabo98** [comment]
Here you go.  By default it installs as the current user.  When it's not working, I can't type at all.  Any idea why wvd hangs on "please wait for the windows search" on logout?

![image](https://user-images.githubusercontent.com/28962523/140976938-ce32b902-6b8d-4ae8-b7f9-b2947938b202.png)

**Comment by daxian-dbw** [comment]
> By default it installs as the current user. When it's not working, I can't type at all.

Once you moved to PS 7.2 + PSReadLine 2.2.0-beta4, typing works as normal, right? I'd love to get your confirmation on this 😄
I marked this issue with `Resolution-Not Repro` since it seems working fine with the latest versions of both PS and PSReadLine.
Please let me know this is not the case.

> Any idea why wvd hangs on "please wait for the windows search" on logout?

Hmm, no idea 😕, but that sounds like something specific to the windows virtual desktop itself.

**Comment by daxian-dbw** [comment]
This issue is actually fixed by https://github.com/PowerShell/PSReadLine/pull/2632, which was first shipped in 2.2.0-beta4.
